pkg/sentry/arch/signal_arm64.go

```diff
@@ -115,12 +116,27 @@ func (c *context64) SignalSetup(st *Stack, act *SignalAct, info *SignalInfo, alt
 	c.Regs.Regs[0] = uint64(info.Signo)
 	c.Regs.Regs[1] = uint64(infoAddr)
 	c.Regs.Regs[2] = uint64(ucAddr)
-
```

Comment:
You should pass the address of the VDSO either directly or through a context rather than stuffing it in an otherwise unused field. The latter is clever, but not easy to read.

Reply:
Done.